src/MessageBox.Avalonia.Example/MainWindow.xaml.cs

```diff
@@ -14,20 +12,44 @@ public class MainWindow : Window
         public MainWindow()
         {
             InitializeComponent();
-
+            var messageBoxCustomWindow = MessageBox.Avalonia.MessageBoxManager.GetMessageBoxHyperlinkWindow(new MessageBoxHyperlinkParams()
+            {
+                CanResize = true,
+                Style = MessageBoxAvaloniaEnums.Style.MacOs,
+                HyperlinkContentProvider = new[]{
+                    new HyperlinkContent { Alias = "dedede         ", Url = "https://avaloniaui.net/docs/styles/styles" },
+                    new HyperlinkContent { Alias="edvyydebbvydebvyed         "},
+                    new HyperlinkContent { Url= "https://avaloniaui.net/docs/styles/styles" }
+                },
+                WindowStartupLocation = WindowStartupLocation.CenterScreen,
+                ButtonDefinitions = MessageBoxAvaloniaEnums.ButtonEnum.Ok
+            });
+            messageBoxCustomWindow.Show();
             this.FindControl<Button>("btnClick").Click += MainWindow_Click;
         }
 
         private async void MainWindow_Click(object sender, RoutedEventArgs e)
         {
-            var messageBoxCustomWindow = MessageBox.Avalonia.MessageBoxManager.GetMessageBoxInputWindow(new MessageBoxInputParams() {
-                ContentMessage = "Message",
-                ButtonDefinitions = new []{new ButtonDefinition{Name = "My"},new ButtonDefinition{Name = "Buttons",Type = MessageBoxAvaloniaEnums.ButtonType.Colored} },
+            var messageBoxCustomWindow = MessageBox.Avalonia.MessageBoxManager.GetMessageBoxStandardWindow(new MessageBoxStandardParams()
+            {
+                CanResize = true,
+                WindowStartupLocation = WindowStartupLocation.CenterScreen,
+                ButtonDefinitions = MessageBoxAvaloniaEnums.ButtonEnum.Ok
+            });
+            var messageBoxCustomWindow2 = MessageBox.Avalonia.MessageBoxManager.GetMessageBoxInputWindow(new MessageBoxInputParams()
+            {
+                CanResize = true,
                 WindowStartupLocation = WindowStartupLocation.CenterScreen
-               
             });
+            //var messageBoxCustomWindow2 = MessageBox.Avalonia.MessageBoxManager.GetMessageBoxStandardWindow(new MessageBoxStandardParams()
```

Comment:
Why do we need to have the code commented out?

Reply:
@rstm-sf MainWindow it's only for testing so I don't think wee need to keep code clean here. But if it's needed I may do a cleanup

Reply:
This is a sample, so leaving islands of incomprehensible lines is not worth it